### ISSUE TITLE
Adding midpoints during draw / validation gestures causes stack trace

### ISSUE THREAD

**Issue by Bonkles** [facebookincubator/RapiD#647]
Stack trace generated: 

```
Uncaught TypeError TypeError: Cannot read properties of undefined (reading 'id')
    at <anonymous> (/Users/clarkben/working/RapiD/modules/actions/add_midpoint.js:10:54)
    at _act (/Users/clarkben/working/RapiD/modules/core/history.js:43:29)
    at _perform (/Users/clarkben/working/RapiD/modules/core/history.js:61:28)
    at perform (/Users/clarkben/working/RapiD/modules/core/history.js:220:24)
    at <anonymous> (/Users/clarkben/working/RapiD/modules/core/context.js:318:25)
    at _clickWay (/Users/clarkben/working/RapiD/modules/modes/ModeAddPoint.js:135:18)
    at _click (/Users/clarkben/working/RapiD/modules/modes/ModeAddPoint.js:107:14)
    at emit (/Users/clarkben/working/RapiD/node_modules/eventemitter3/index.js:181:35)
    at _doClick (/Users/clarkben/working/RapiD/modules/behaviors/BehaviorDraw.js:303:10)
    at _pointerup (/Users/clarkben/working/RapiD/modules/behaviors/BehaviorDraw.js:213:12)
    at emit (/Users/clarkben/working/RapiD/node_modules/eventemitter3/index.js:202:33)
    at _pointerup (/Users/clarkben/working/RapiD/modules/pixi/PixiEvents.js:242:10)
    at notifyListeners (/Users/clarkben/working/RapiD/node_modules/@pixi/events/src/EventBoundary.ts:1313:26)
    at notifyTarget (/Users/clarkben/working/RapiD/node_modules/@pixi/events/src/EventBoundary.ts:545:14)
    at propagate (/Users/clarkben/working/RapiD/node_modules/@pixi/events/src/EventBoundary.ts:336:18)
    at dispatchEvent (/Users/clarkben/working/RapiD/node_modules/@pixi/events/src/EventBoundary.ts:240:14)
    at mapPointerUp (/Users/clarkben/working/RapiD/node_modules/@pixi/events/src/EventBoundary.ts:830:14)
    at mapEvent (/Users/clarkben/working/RapiD/node_modules/@pixi/events/src/EventBoundary.ts:261:28)
    at onPointerUp (/Users/clarkben/working/RapiD/node_modules/@pixi/events/src/EventSystem.ts:290:31)
```

**Comment by Bonkles** [comment]
Looks like this problem crept in with my fix to #538.

**Comment by Bonkles** [comment]
Fixed in `018c9ab0c`.
